app/views/shared/_result_information.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="govuk-accordion__section">
+<div class="govuk-accordion__section result_information">
   <div class="govuk-accordion__section-header">
     <h3 class="govuk-accordion__section-heading">
       <button type="button" id="accordion-default-heading-2" aria-controls="accordion-default-heading-3" class="govuk-accordion__section-button" aria-expanded="false">
```

Comment:
I've just noticed that the id and aria-control id are different here. I must have missed it in the last PR 😬

Reply:
Badly copy-pasted from elsewhere and forgot to update, sorry 😬